wpilib/wpilib/buttons/trigger.py

```diff
@@ -133,15 +136,13 @@ def execute():
         from ..command import Scheduler
         Scheduler.getInstance().addButton(execute)
 
-    def getSmartDashboardType(self):
-        """These methods continue to return the "Button" :class:`.SmartDashboard` type
-        until we decided to create a Trigger widget type for the dashboard.
-        """
-        return "Button"
-
-    def initTable(self, table):
-        if table is not None and isinstance(table, NetworkTable):
-            self.pressedEntry = table.getEntry("pressed")
-            self.pressedEntry.setBoolean(self.get())
-        else:
-            self.pressedEntry = None
+    def _safeState(self) -> None:
+        self.sendablePressed = False
+
+    def _setPressed(self, value: bool) -> None:
+        self.sendablePressed = value
+
+    def initSendable(self, builder: SendableBuilder):
```

Comment:
I actually don't think it's worth the trouble to type hint this method. We don't expect teams to call `initSendable` directly, so we don't actually need to document it. (sphinx-autodoc-typehints will create a docstring on type hinted functions if there isn't one already.)